### PR TITLE
RSSFE-06 WIP add edit profile form and updateUser mutation

### PR DESCRIPTION
**Pull request by self067**
1. Poppins font
2. colors constant
3. styled components
  TextRegular, TextMedium,  TextSemiBold,  TextBold    - css styles with  widths 400,500,600,700, font Poppins 16/24px  
  PageTitle,  Button,  Label,  Input  - basic styled components

### CODE REVIEW

**Comment by self067** [comment]
4. add Header component

File: src/components/App/index.tsx [hunk 1]
```diff
@@ -9,9 +9,10 @@ import {
   NotFoundPage,
   EditProfile,
 } from 'modules';
-import { Loader, PrivateRoute } from 'components';
+import { Loader, PrivateRoute, Header } from 'components';
 import { selectToken } from 'modules/LoginPage/selectors';
 import { AUTH_TOKEN, SET_TOKEN } from 'appConstants';
+import { GlobalStyle } from 'typography';
```
Comment (MadaShindeInai) [inline src/components/App/index.tsx:15]: please write globalStyles in src/typography/common.css

File: src/components/App/index.tsx [hunk 2]
```diff
@@ -27,28 +28,32 @@ export const App: FC = () => {
...
-        path="/editProfile"
-        isLoggedIn={!!loginToken}
-        component={EditProfile}
-      />
-      <Route exact path="/token/:id" component={TokenPage} />
-      <Route exact path="/login" component={LoginPage} />
-      <Route path="*" component={NotFoundPage} />
-    </Switch>
+    <>
+      <GlobalStyle />
```
Comment (MadaShindeInai) [inline src/components/App/index.tsx:32]: remove and add this styles to src/typography/common.css.

File: src/typography/common.css [hunk 3]
```diff
@@ -1,6 +1,8 @@
 body {
   margin: 0;
-  font-family: "SF UI Text", sans-serif;
+  font-family: "Poppins", sans-serif;
```
[the remark this answers is not in the document]
Comment (MadaShindeInai) [reply src/typography/common.css:3]: add styles from GlobalStyles there.

File: src/typography/index.ts [hunk 4]
```diff
@@ -1,6 +1,26 @@
...
 import { Link } from 'react-router-dom';
 import { ReactComponent as SVGIcon } from '../assets/svg/logo.svg';
+import { ReactComponent as RSLogoIcon } from '../assets/svg/rslogo.svg';
+import {
+  WHITE_COLOR,
+  BACKGROUND_COLOR,
+  MAIN1_COLOR,
+  DARK_TEXT_COLOR,
+  LIGHT_TEXT_COLOR,
+} from '../appConstants/colors';
```
Comment (MadaShindeInai) [inline src/typography/index.ts:11]: you should import  as `} from 'appConstants/colors';`

File: src/typography/index.ts [hunk 5]
```diff
@@ -1,6 +1,26 @@
-import styled from 'styled-components';
+import styled, { css, createGlobalStyle } from 'styled-components';
```
Comment (MadaShindeInai) [inline src/typography/index.ts:1]: we will not use globalStyled in this project. it can cause problems later.

File: src/typography/index.ts [hunk 6]
```diff
@@ -1,6 +1,26 @@
...
+import { ReactComponent as RSLogoIcon } from '../assets/svg/rslogo.svg';
+import {
+  WHITE_COLOR,
+  BACKGROUND_COLOR,
+  MAIN1_COLOR,
+  DARK_TEXT_COLOR,
+  LIGHT_TEXT_COLOR,
+} from '../appConstants/colors';
+
+export const GlobalStyle = createGlobalStyle`
```
Comment (MadaShindeInai) [inline src/typography/index.ts:13]: remove and add this styles to common.css

File: src/typography/index.ts [hunk 7]
```diff
@@ -1,6 +1,26 @@
...
+  }
+`;
+
+interface StyledTextProps {
+  color?: string;
+  fontSize?: string;
+  lineHeight?: string;
+}
 
 export const LinkText = styled(Link)`
```
Comment (MadaShindeInai) [inline src/typography/index.ts:25]: we can remove this now as unused. it was example

File: src/typography/index.ts [hunk 8]
```diff
@@ -11,3 +31,57 @@ export const Logo = styled(SVGIcon)`
   width: 50px;
   height: 50px;
 `;
+
+export const RSLogo = styled(RSLogoIcon)`
+  width: 84px;
+  height: 30px;
+`;
+
+const StyledText = css<StyledTextProps>`
```
Comment (MadaShindeInai) [inline src/typography/index.ts:40]: wow! newer used stc this way. Cool! Maybe we can write components and css declarations in different files ( daclarations with `css` in separate file, and import it in file with `styled.`  as components)?

**Review by MadaShindeInai — CHANGES_REQUESTED** [review]

File: src/components/Header/index.tsx [hunk 9]
```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { RSLogo } from 'typography';
+import { StyledHeader } from './styled';
+import { Container } from 'components/Container';
```
[the remark this answers is not in the document]
Comment (self067) [reply src/components/Header/index.tsx:5]: пока перенес в styled, если ещё кому то понадобится тогда сделаем общий файл

File: src/modules/EditProfile/index.tsx [hunk 10]
```diff
@@ -1,16 +1,61 @@
...
 // import { useSelector } from 'react-redux';
 // import { useCoursesQuery } from 'hooks/graphql';
 // import { selectUserData } from 'modules/StudentsTable/selectors';
-import { Logo } from 'typography';
+import { PageTitle, Button } from 'typography';
+import { InputField } from 'components/InputField';
+import { SelectField } from 'components/SelectField';
+import { WHITE_COLOR } from 'appConstants/colors';
+
+const EditProfileWrapper = styled.div`
```
[the remark this answers is not in the document]
Comment (self067) [reply src/modules/EditProfile/index.tsx]: перенес, видимо не запушил

**Review by MadaShindeInai — APPROVED** [review]

**Pull request merged**